lib/elixir/lib/module/types/descr.ex

```diff
@@ -188,6 +244,49 @@ defmodule Module.Types.Descr do
 
   defp iterator_difference(:none, map), do: map
 
+  ## Type relations
+
+  @doc """
+  Check if a type is a subtype of another.
+
+  If     `left  ~= (left_dyn  and dynamic()) or left_static`
+  and    `right ~= (right_dyn and dynamic()) or right_static`
```

Comment:
What does the `~=` operator mean? Is it just "assignment"? Shall we use `=` instead?

Reply:
Yes, we can use `=`. I used it to mean "are those `equal?`" and differentiate it from structural equality.